scripts/get-icons/index.ts

```diff
@@ -0,0 +1,95 @@
+import axios from 'axios';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+
+const ICON_FILE = 'Ai7AELHC6KCz38qKZkvuHo';
+
+const ICON_FRAME = 'UI icons 24(w)x24(w)';
+
+const OUTPUT_DIR = '../../packages/@guardian/src-icons/svg';
```

Comment:
Sorry, on thinking about it, would it be better to pluralise this directory name? We tend to pluralise things elsewhere in this repo (`components`, `foundations`, `icons`, `'libs`)

```suggestion
const OUTPUT_DIR = '../../packages/@guardian/src-icons/svgs';
```